theoplayer/how-to-guides/01-ads/07-spotx.md

```diff
@@ -45,7 +44,7 @@ the `{{SpotXAdDescription}}` object provides the following properties:
 
 Example:
 
-```js
```

Comment:
Huh, does `js` not work? 😕

Reply:
Both `js` and `javascript` work. 

Not necessary to make this change actually. I think I was just going to edit the snippet here but then left it after going through it. 😅

Reply:
I'm fine with either, I was just worried that something broke horribly with our code blocks. 😛